editoast/src/views/timetable/stdcm.rs

```diff
@@ -127,6 +128,13 @@ pub(in crate::views) struct InfraIdQueryParam {
 /// If the simulation fails, the function uses a virtual train to detect conflicts
 /// with existing train schedules. It then returns both the conflict information
 /// and the pathfinding result from the virtual train's simulation.
+#[tracing::instrument(
+    target = "editoast::timetable",
+    name = "stdcm",
+    skip_all,
+    err,
+    fields(request)
+)]
```

Comment:
I am not sure how adding instrumentation to an endpoint will behave. Endpoints are traced by default.

@woshilapin perhaps you can enlighten me on this.

Reply:
It creates a new trace, which is itself part of the endpoint trace. The new trace *only* contains this function, while the endpoint trace seems to have *a little* overhead at the end. 

<img width="763" height="402" alt="image" src="https://github.com/user-attachments/assets/740ba2a5-990d-4e2a-bfbf-1ae52de0d29b" />